cxx-checks/src/main/java/org/sonar/cxx/checks/FunctionComplexityCheck.java

```diff
@@ -46,7 +47,8 @@
   offset = "10min",
   effortToFixDescription = "per complexity point above the threshold")
 public class FunctionComplexityCheck extends SquidCheck<Grammar> {
-  private static final int DEFAULT_MAX = 10;
+
+  private static final int DEFAULT_MAX = 8;
```

Comment:
My understanding is:
1-10 = Low risk program
11-20 = Moderate risk
21-50 = High risk
>50 = Most complex and highly unstable method
Especially in OO development inside of methods 8 is a good value.
Maybe we should allow to define the rule more than once? So someone can e.g. set 8 as minor and 21 as critical?

> Am 30.06.2015 um 19:12 schrieb Jorge Costa <notifications@github.com>:
> 
> In cxx-checks/src/main/java/org/sonar/cxx/checks/FunctionComplexityCheck.java:
> 
> > @@ -46,7 +47,8 @@
> >    offset = "10min",
> >    effortToFixDescription = "per complexity point above the threshold")
> >  public class FunctionComplexityCheck extends SquidCheck<Grammar> {
> > -  private static final int DEFAULT_MAX = 10;
> > +
> > +  private static final int DEFAULT_MAX = 8;
> any good reason to change this to 8... we usually use 12
> 
> —
> Reply to this email directly or view it on GitHub.
>